### DISCUSSION TITLE
Crash on Arriving halfway point

### DISCUSSION THREAD

**Discussion by Hi99simon** [eopeter/flutter_mapbox_navigation#259] [Q&A]
Hi Community!

I used mapbox navigation plugin,  successfully implemented, but problem occured when using multiple point for routes.

For example, adding 3 waypoints to build route, A,B,C point.

The navigation can build the route, But when arriving point B, the plugin will cause the app crash, suddenly quit app.

**Comment by eopeter** [comment]
Is this on Android or iOS or both?

**Comment by Hi99simon** [comment]
I tested on Ios, not yet tested on android.

I think some native code not handled properly?

「Emmanuel Oche ***@***.***>」在 2023年6月2日 週五，下午11:44 寫道：
